### DISCUSSION TITLE
Instant Commands panel

### DISCUSSION THREAD

**Discussion by lucasjinreal** [flyerhq/flutter_chat_ui#158] [Ideas & Requests]
Hi, just want is there a simple way current make a Instant Command panel currently?

![image](https://user-images.githubusercontent.com/21303438/141669987-597ef003-9fa5-4766-ac88-f39f10c02b31.png)

this is really userful when we want apply some commands like telegram

**Comment by demchenkoalex** [comment]
No I don't think so. How we can generalise that? Since there might be tons of actions for different use cases. Should it be like an empty pop up on top of the input with an array of widget builders or something?

**Comment by lucasjinreal** [comment]
@demchenkoalex Yes. that's what i mean. Since commands are fully customized. But users can impment that layout of blank popout view.

Currently I using flychat implement a ChatAPP which want a **COMMANDS** features, like I want:

- /invite: invite someone to group;
- /menu: show menu in this group;

So that whole chatting screen becomes more functional.

**Reply by demchenkoalex** [reply]
Maybe you want to add a PR? Looks like we need to do an empty popup which accepts a list of widgets
